### DISCUSSION TITLE
Is Joomla!3.10.12 against guard to vulnerability?

### DISCUSSION THREAD

**Discussion by OscarLeung19920222** [joomla/joomla-cms#42644] [General] [answered]
As far as I know, the eol of 3.10.12 has been reached on Augusts 2023.
My client refused to upgrade to J4 as the cost and the change of admin interface.
Regardless to client's thought, my question is:
Is J3 eol means it is vulnerable? What if the existing source code is maintenance and support by my own?
Does the self-maintain version against any vulnerability policy by, e.t.c. owasp or cwe?
Can I edit the Joomla project version to my own preoperatory version?

Any idea or thought is truly appreciated, I am so confused in this area.
Thanks so much for your reply.

**Comment by sba** [comment, marked as answer]
There ist an Extended Long Term Support (eLTS) available, please read the announcement here:
https://www.joomla.org/announcements/release-news/5894-joomla-is-18-and-extended-security-support-for-joomla-3.html
https://elts.joomla.org/

**Reply by OscarLeung19920222** [reply]
Thank you for the information, it really helps a lot to answer my question.
One thing I would like to know that is 3.10.12 supported PHP 8.1 or later ?

**Reply by sba** [reply]
I have a website on Joomla 3.10.12 running with php 8.1. Maybe you need to turn off deprecated warnings.
